atlasdb-client/src/main/java/com/palantir/atlasdb/schema/SweepSchema.java

```diff
@@ -39,11 +40,14 @@ public Schema get() {
     private static Schema generateSchema() {
         Schema schema = new Schema("Sweep",
                 SweepSchema.class.getPackage().getName() + ".generated",
-                NAMESPACE);
+                NAMESPACE,
+                OptionalType.JAVA8);
```

Comment:
Hmm, good point. Actually, I did this because the generated schema files were already Java 8 for some reason and did not match up with the versions here (and I thought we should be consistent)!

@amarzoca for SA - I think you might have run into this in #2210.